lib/firespring_dev_commands/version.rb

```diff
@@ -6,6 +6,6 @@ module Version
     # Use 'v.v.v.pre.alpha.v' for pre-release vesions
     # Use 'v.v.v.beta.v for beta versions
     # Use semantic versioning for any releases (https://semver.org/)
-    VERSION = '1.4.4'.freeze
+    VERSION = '1.4.5'.freeze
```

Comment:
You'll likely want update the version to follow the scheme provided in the comments. You are also going to need to update the remote gem with an updated version number. https://rubygems.org/gems/firespring_dev_commands

`1.4.6.beta.0`

Reply:
Updated